infra/terraform/secure_pipelines/mob_sts_mock_pipeline.tf

```diff
@@ -37,12 +37,12 @@ locals {
       IncludePromotion = "No"
 
       ContainerSignerKmsKeyArn = one(data.aws_cloudformation_stack.container_signer_dev[*].outputs["ContainerSignerKmsKeyArn"])
-      RequireTestContainerSignatureValidation = "Yes"
+      #RequireTestContainerSignatureValidation = "Yes"
```

Comment:
Please may we add a comment to explain why it's commented out so we have context in future to all these lines?

i.e. Disabled until the sts-mock stack pushes a test image to ECR

Reply:
Yep good point